### PR TITLE
Bugfix/rjxs/observables

### PR DESCRIPTION
**Pull request by MihlaliMgwebi**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,20 +8,8 @@
...
-  <!-- Include Leaflet JavaScript file after Leaflet’s CSS: -->
-  <script src="https://unpkg.com/leaflet@1.9.3/dist/leaflet.js"
-    integrity="sha256-WBkoXOwTeyKclOHuWtc+i2uENFpDZ9YPdf5Hf+D7ewM=" crossorigin=""></script>
-
-  <!-- Include axios to consume API -->
-  <script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
-
   <!-- Include external stylesheet because dom manip keeps overriding -->
-  <link rel="stylesheet" href="style.scss">
+  <!-- <link rel="stylesheet" href="style.scss"> -->
```
Comment (CharelMarais) [inline index.html:12]: You should remove any unused code comments

File: src/services/dom.service.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+    });
+    return flightSummaryCollapsibleButton;
+}
+
+
+// STEP 4.3: Render card
+function createFlightDetailsCard(flight: IFlight) {
+    const flightDetailsCard = document.createElement("div");
+    flightDetailsCard.id = `flight__details-${flight.icao24}`;
+    // flightDetailsCard.className = `flight__details hide`;
```
Comment (CharelMarais) [inline src/services/dom.service.ts:93]: Remove any unused code comments

File: src/vite-env.d.ts [hunk 3]
```diff
@@ -0,0 +1 @@
+/// <reference types="vite/client" />
```
Comment (CharelMarais) [inline src/vite-env.d.ts:1]: What is the reason for this file

**Review by CharelMarais — APPROVED** [review]
Code looks good, please check my comments for my suggestions

File: src/vite-env.d.ts (hunk 3 above)
Comment (MihlaliMgwebi) [reply src/vite-env.d.ts:1]: Will remove. I was going to store my base url here.

File: src/services/dom.service.ts (hunk 2 above)
Comment (MihlaliMgwebi) [reply src/services/dom.service.ts:93]: Thank you for the feedback. I will remove the comments and file then merge into develop.

File: index.html (hunk 1 above)
Comment (MihlaliMgwebi) [reply index.html:12]: removed. Thank you

File: src/services/dom.service.ts (hunk 2 above)
Comment (MihlaliMgwebi) [reply src/services/dom.service.ts:93]: removed, thank you.

**Comment by MihlaliMgwebi** [comment]
Removed dead code and unused file. Approved and merging changes.

File: app/components/timeInput/timeInput.ts [hunk 4]
```diff
@@ -53,24 +52,19 @@ export function appendTimeInputToParent(parentElement: HTMLDivElement): void {
...
-    return {
-        next: (event) => {
-            if (event && event.target) {
-                const timeHTMLInputElement: HTMLInputElement = event.target as HTMLInputElement;
-                const dateTime: string = timeHTMLInputElement.value;
-                const localUnixTimestampInSeconds = Utils.convertDateTimeToLocalUnixTimestampInSeconds(dateTime);
-                getFirst20FlightDetails(localUnixTimestampInSeconds);
-                return localUnixTimestampInSeconds;
-            }
+// function getTimeInMillisecondsOnTimeInputEvent(): Observable<number> {
```
Comment (ntokozo-nhlangulela) [inline app/components/timeInput/timeInput.ts]: Don't forget to remove the commented code before submission.

File: src/vite-env.d.ts (hunk 3 above)
Comment (ntokozo-nhlangulela) [reply src/vite-env.d.ts]: Don't forget to remove it then.

**Pull request merged**